setup.cfg

```diff
@@ -19,17 +19,22 @@ install_requires =
 spacy_architectures =
     spacy-experimental.Bilinear.v1 = spacy_experimental.biaffine_parser.bilinear:build_bilinear
     spacy-experimental.PairwiseBilinear.v1 = spacy_experimental.biaffine_parser.pairwise_bilinear:build_pairwise_bilinear
+    spacy-experimental.span_finder_model.v1 = spacy_experimental.span_finder.span_finder_model:build_finder_model
```

Comment:
The architectures usually have names like `spacy.TextCatEnsemble.v2` or `spacy.Tagger.v1` and don't include `model `in the name, so something like `spacy-experimental.SpanFinder.v1`.